sections/what_is_rust.tex

```diff
@@ -8,32 +8,53 @@ \section{What is Rust?}
 		that runs blazingly fast, prevents nearly all segfaults,\\
 		and guarantees thread safety.»\\
 		\vspace{0.5em}
-		{\normalfont \small --- www.rust-lang.org}
+		{\normalfont \small --- www.rust-lang.org (until 2018)}
+	\end{quote}
+
+	\vspace{2em}
+
+	\begin{quote}
+		«A language empowering everyone\\
+		to build reliable and efficient software.»\\
+		\vspace{0.5em}
+		{\normalfont \small --- www.rust-lang.org (since 2019)}
 	\end{quote}
 \end{frame}
 
 %%%
 
-\begin{frame}{What's wrong with systems languages?}
-	\begin{itemize}
-		\item It's difficult to write secure code.
-		\item It's very difficult to write multithreaded code.
-	\end{itemize}
-	These are the problems Rust was made to address.
+\begin{frame}{Main Claims}
+	\textbf{Performance}:\\Rust is \textbf{blazingly fast and memory-efficient}:
+	with \textbf{no runtime or garbage collector}, it can power
+	performance-critical services, run on embedded devices, and easily integrate
+	with other languages.
```

Comment:
Hmm, "other languages" meaning C, right? It isn't directly integrateable with other languages, is it?

Reply:
It's quite easily integrateable with Python and Java due to good and easy-to-use libraries (which are based on good C compatibility, yes).

The quote is taken from the website, so it's probably fair to keep it.